src/app/tests/suites/certification/ci-pics-values

```diff
@@ -2,6 +2,7 @@ APPDEVICE.S=0
 MCORE.BRIDGECLIENT=0
 MCORE.DT_SW_COMP=0
 MCORE.UI.FACTORYRESET=1
+MCORE.Afffa=0
```

Comment:
Is there any way we can guarantee that this stays in sync with the compile-time EventList support?